src/layouts/Desktop/LogInLayout/LogInLayout.tsx

```diff
@@ -24,7 +24,7 @@ const LogInLayout = () => {
       <div className='log-in-right'>
         <div className='log-in-right-header'>
           Don&#39;t have an account?
-          <Button onClick={() => navigate('/singin')} variant={ButtonVariant.Default}>
+          <Button onClick={() => navigate('/signin')} variant={ButtonVariant.Default}>
```

Comment:
i have an idea, we could create constants, for all routes, and use this constants here, this will add you posibility to change routing  just by changing one variable  instead of searching across the project to change all routes